Makefile

```diff
@@ -61,8 +65,10 @@ shellcheck:
 # problematic lines are conveyed to the user so they can be fixed.
 #
 shfmtcheck:
-	! shfmt -d $$(find scripts -type f -executable) \
-		$$(find live-build/misc/live-build-hooks -type f -executable) | grep .
+	! shfmt -d $$(find scripts -type f $(FINDEXEC)) \
+		$$(find live-build/misc/live-build-hooks -type f $(FINDEXEC)) | grep .
 
 ansiblecheck:
 	ansible-lint $$(find bootstrap live-build/variants -name playbook.yml)
+
+checks: shellcheck shfmtcheck ansiblecheck
```

Comment:
I think the normal convention is to use `make check` rather than `make checks`..

alternatively, I see ZOL has a `make checkstyle` target; I'd be OK using that to be consistent with another codebase we'll frequently be using.